# Client sends nothing
---
bool success #success response (true)